src/Elcodi/Bridge/PaymentSuiteBridgeBundle/Services/PaymentBridge.php

```diff
@@ -61,18 +62,21 @@ class PaymentBridge implements PaymentBridgeInterface
     private $currencyConverter;
 
     /**
-     * @param OrderRepository   $orderRepository   Order repository
-     * @param CartWrapper       $cartWrapper
-     * @param CurrencyConverter $currencyConverter
+     * @param OrderRepository         $orderRepository         Order repository
+     * @param CartWrapper             $cartWrapper
+     * @param CurrencyConverter       $currencyConverter
+     * @param PurchasableNameResolver $purchasableNameResolver
      */
     public function __construct(
         OrderRepository $orderRepository,
         CartWrapper $cartWrapper,
-        CurrencyConverter $currencyConverter
+        CurrencyConverter $currencyConverter,
+        PurchasableNameResolver $purchasableNameResolver
     ) {
         $this->orderRepository = $orderRepository;
         $this->cartWrapper = $cartWrapper;
         $this->currencyConverter = $currencyConverter;
+        $this->purchasableNameResolver = $purchasableNameResolver;
```

Comment:
This parameter is not being assigned anywhere.
You should add the local variable as well (private)